README.md

```diff
@@ -203,76 +147,92 @@ const myNaming = bemNaming({
                                   // element and modifier's name
 
 myNaming.parse('block--mod_val'); // { block: 'block',
-                                  //   modName: 'mod', modVal: 'val' }
+                                  //   mod: { name: 'mod', val: 'val' } }
 
-myNaming.stringify({              // 'blockName-elemName--boolElemMod'
+myNaming.stringify({              // 'blockName-elemName--simpleElemMod'
     block: 'blockName',
     elem: 'elemName',
-    modName: 'boolElemMod'
+    mod: 'simpleElemMod'
 });
 ```
 
-### bemNaming({ elem, mod, wordPattern })
-
-#### elem
-
-Type: `String`
-
-Default: `__`
-
-Separates element's name from block
-
-#### mod
-
-Type: `String`, `{ name: String, val: String }`
-
-Default: `_`
+API
+---
 
-Separates modifiers from blocks and elements.
+* [bemNaming({ elem, mod, wordPattern })](#bemnaming-elem-mod-wordpattern-)
+* [parse(str)](#parsestr)
+* [stringify(obj)](#stringifyobj)
+* [elemDelim](#elemdelim)
+* [modDelim](#moddelim)
+* [modValDelim](#modvaldelim)
 
-This option can take object with following fields:
+### bemNaming({ elem, mod, wordPattern })
 
-* `name` — separates name of modifier from blocks and elements.
+Parameter     | Type     | Description                                                                | Default
+--------------|----------|----------------------------------------------------------------------------|---------
+`elem`        | `string` | Separates element's name from block.                                       | `__`
+`mod`         | `string`, `{ name: string, val: string }` | Separates modifier from block or element. | `_`
+`mod.name`    | `string` | Separates name of modifier from block or element.                          | `_`
+`mod.val`     | `string` | Separates value of modifier from name of modifier.                         | Default as the value of the `mod.name`.
+`wordPattern` | `string` | Defines which symbols can be used for block, element and modifier's names. | `[a-z0-9]+(?:-[a-z0-9]+)*`
```

Comment:
Defines which characters can be used in names of blocks, elements, and modifiers.